interfaces/builtin/login_session_control.go

```diff
@@ -38,28 +38,25 @@ dbus (send,receive)
     bus=system
     path=/org/freedesktop/login1/{seat,session}/**
     interface=org.freedesktop.DBus.Properties
-    member={GetAll,PropertiesChanged,Get}
```

Comment:
Please do not remove this line as it then allows to `Set` the various properties directly.

Reply:
OTOH, this is only for the `seat` and `session` objects (judging from their path), so I guess the client should be able to alter them?

Reply:
Perhaps - I am just worried that modifying the properties directly may not be safe, and that it is better to use the various methods of the interfaces to do this instead.

Reply:
Maybe we can leave it out of this PR and include it if there's a use case in the future?

Reply:
Sounds good to me!

Reply:
Done!